Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -34,7 +34,17 @@ MemoryViewWidget::MemoryViewWidget(QWidget* parent) : QTableWidget(parent)
 
   setFont(Settings::Instance().GetDebugFont());
 
-  connect(&Settings::Instance(), &Settings::DebugFontChanged, this, &QWidget::setFont);
+  const QFontMetrics fm(Settings::Instance().GetDebugFont());
+  m_font_vspace = fm.lineSpacing();
+  m_font_width = fm.averageCharWidth();
+
+  connect(&Settings::Instance(), &Settings::DebugFontChanged, this, [this] {
+    const QFontMetrics fmet(Settings::Instance().GetDebugFont());
+    m_font_vspace = fmet.lineSpacing();
+    m_font_width = fmet.averageCharWidth();
+    setFont(Settings::Instance().GetDebugFont());
```

Comment:
This is practically the same code twice -- extract that to a function, maybe?

Reply:
Yeah, that would work.